app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/TimeSegment.java

```diff
@@ -1,36 +1,30 @@
 package nerd.tuxmobil.fahrplan.congress.schedule;
 
-import android.text.format.Time;
-
-import java.util.Locale;
+import info.metadude.android.eventfahrplan.commons.Clock;
+import info.metadude.android.eventfahrplan.commons.DateFormatter;
+import info.metadude.android.eventfahrplan.commons.SystemClock;
 
 class TimeSegment {
-
-    private static final String HOUR_MINUTE_DIVIDER = ":";
-    private static final String TIME_PATTERN = "%02d";
     private static final int MINUTES_PER_HOUR = 60;
 
     private final int hour;
     private final int minute;
+    private int minutesOfTheDay;
 
     TimeSegment(int minutesOfTheDay) {
         hour = minutesOfTheDay / MINUTES_PER_HOUR;
         minute = minutesOfTheDay % MINUTES_PER_HOUR;
+        this.minutesOfTheDay = minutesOfTheDay;
     }
 
     String getFormattedText() {
-        StringBuilder stringBuilder = new StringBuilder();
-        String formattedHour = String.format(Locale.US, TIME_PATTERN, hour);
-        String formattedMinute = String.format(Locale.US, TIME_PATTERN, minute);
-        return stringBuilder
-                .append(formattedHour)
-                .append(HOUR_MINUTE_DIVIDER)
-                .append(formattedMinute)
-                .toString();
+        SystemClock clock = new SystemClock().startOfDay();
+        clock.plusSeconds(minutesOfTheDay * 60);
```

Comment:
# Encapsulation
Let's use `clock.plusMinutes(minutesOfTheDay);` instead of `clock.plusSeconds(minutesOfTheDay * 60);`

Reply:
Done in 23d7e05a3226ef2860144ab3fb7222db036a8fe0. Thanks.